spark/src/test/scala/org/apache/spark/sql/delta/DeletionVectorsTestUtils.scala

```diff
@@ -127,27 +128,53 @@ trait DeletionVectorsTestUtils extends QueryTest with SharedSparkSession with De
       dataDF: DataFrame,
       partitionBy: Seq[String] = Seq.empty,
       enableDVs: Boolean = true,
-      conf: Seq[(String, String)] = Nil)
+      conf: Seq[(String, String)] = Nil,
+      saveAsTable: Boolean = false)
       (fn: (() => io.delta.tables.DeltaTable, DeltaLog) => Unit): Unit = {
-    withTempPath { path =>
-      val tablePath = new Path(path.getAbsolutePath)
+    def createTable(
+       saveAsTable: Boolean,
+       tableNameOpt: Option[String],
+       tablePathOpt: Option[Path]): Unit = {
+      val tableName = tableNameOpt.getOrElse("")
+      val tablePath = tablePathOpt.getOrElse(new Path("."))
```

Comment:
Can we directly use `null` here?

E.g.,
```suggestion
      val tablePath = tablePathOpt.getOrElse(null)
```